### PR TITLE
Add Python 3 installation instructions

### PR DESCRIPTION
**Pull request by mmcdaris**
Adds Python 3 installation instructions to the Python doc.

### CODE REVIEW

File: jekyll/_docs/installing-airbrake/installing-airbrake-in-a-python-app.md [hunk 1]
```diff
@@ -15,14 +15,69 @@ description: installing airbrake in a Python app
...
 
-### Download the Airbrake notifier
+- Using a 3.x version of Python? follow the [Python 3 instructions](#python-3x).
+- Using a 2.x version of Python? follow the [Python 2 instructions](#python-27).
+
+## Python 3.x+
+
+### Installation
+
+The pybrake notifier supports Python 3.4 and above.
```
Comment (phumpal) [inline jekyll/_docs/installing-airbrake/installing-airbrake-in-a-python-app.md]: Link to pybrake?

File: jekyll/_docs/installing-airbrake/installing-airbrake-in-a-python-app.md [hunk 2]
```diff
@@ -15,14 +15,69 @@ description: installing airbrake in a Python app
...
+### Taking pybrake further
+Now that pybrake is configured and ready to go we recommend you check out some
+of the additional integrations and features pybrake supports:
+- [integrate with the logging standard library](https://github.com/airbrake/pybrake#logging-integration)
+- [integrating with Django via Airbrake middleware](https://github.com/airbrake/pybrake#django-integration)
+- [integrating with Flask via the blinker library](https://github.com/airbrake/pybrake#flask-integration)
+- [add custom information to notices before they are sent](https://github.com/airbrake/pybrake#usage)
+- [define custom filters to ignore notices by type or any other criteria](https://github.com/airbrake/pybrake#usage)
+
+Visit the [pybrake GitHub repo](https://github.com/airbrake/airbrake-python)
```
Comment (phumpal) [inline jekyll/_docs/installing-airbrake/installing-airbrake-in-a-python-app.md]: s/airbrake-python/pybrake/

File: jekyll/_docs/installing-airbrake/installing-airbrake-in-a-python-app.md (hunk 1 above)
Comment (mmcdaris) [reply jekyll/_docs/installing-airbrake/installing-airbrake-in-a-python-app.md]: Good call, will do!

File: jekyll/_docs/installing-airbrake/installing-airbrake-in-a-python-app.md (hunk 2 above)
Comment (mmcdaris) [reply jekyll/_docs/installing-airbrake/installing-airbrake-in-a-python-app.md]: Nice catch 👍, updating.

**Comment by mmcdaris** [comment]
Thanks for the reviews guys!

**Pull request merged**